### PR TITLE
Guid time

### PR DESCRIPTION
**Pull request by johnthepink**

### CODE REVIEW

File: _source/lib/apollos.coffee [hunk 1]
```diff
@@ -37,10 +37,11 @@ Apollos.user.login = {}
 ###
 Apollos.person = ->
 
-  userDoc = Apollos.user()
+  user = Apollos.user()
 
-  person = Apollos.people.findOne
-    userId: userDoc._id
+  if user.rock?.guid
```
Comment (jbaxleyiii) [inline _source/lib/apollos.coffee]: @johnthepink any reason why we set this with a rock object?

Couldn't we do `Apollos.people.findOne(userGuid: user.guid)`

Then Rock could set that Guid on creation of Apollos people / user from Rock and Apollos could sync that data to Rock and set the Guid of the user / person.

Maybe I'm misunderstanding as well :crying_cat_face:

Comment (johnthepink) [inline _source/lib/apollos.coffee]: @jbaxleyiii we can totally do that. that's just where it was already being set during sync. i move?

Comment (jbaxleyiii) [inline _source/lib/apollos.coffee]: @johnthepink yes, we move, and @benwiley we will need to update the apollos-rock package to support the move

:goat:

**Comment by johnthepink** [comment]
@jbaxleyiii @benwiley does [this](https://github.com/NewSpring/apollos-rock/commit/615cc7805ab7473f91022e79b8d4e26e86f66f33) fix the sync?

**Comment by jbaxleyiii** [comment]
@johnthepink until we get norman and apollos-rock up and running no telling. BUT it look like it should!

**Pull request merged**